src/main/java/core/basesyntax/operations/Buy.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitStorage;
+import core.basesyntax.model.Operation;
+
+public class Buy implements StorageOperation {
```

Comment:
```suggestion
public class PurchaseOperation implements StorageOperation {
```
class name should be noun

Reply:
fixed